spec/features/trainee_actions/adding_trainee_placements_spec.rb

```diff
@@ -6,13 +6,27 @@
   scenario "Add a new placement to an existing trainee" do
```

Comment:
Could we move this spec into `spec/features/form_sections/teacher_training_data/placements`? Generally, any spec covering form stuff should probably go in there. The `trainee_actions` dir is to cover performing actions on the trainee (withdrawing, reinstating etc or at least that was the intention!)